tools/build_cdn.js

```diff
@@ -58,6 +64,22 @@ async function buildCDN(options) {
   log("-----");
 }
 
+
+async function buildSRIDigests(shas) {
+  const temp = await fs.readFile("./tools/templates/DIGESTS.md")
+  const DIGEST_MD = temp.toString()
+
+  const version = require("../package").version
+  const digestList = Object.entries(shas).map((k,v) => `${v} ${k}`).join("\n")
+
+  const out = DIGEST_MD
+    .replace("<!-- $DIGEST_LIST -->", digestList)
+    .replace("<!-- $MIN_JS_DIGEST -->", shas["highlight.min.js"])
+    .replace("<!-- $GO_SHA -->", shas["languages/go.min.js"])
```

Comment:
Why does Go have its own explicit replacement?

Reply:
Cause that's the example language used in the readme.

Reply:
Ha, I hadn't added DIGESTS, added.